FakeNFT/Onboarding/PageViewController.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  PageViewController.swift
+//  FakeNFT
+//
+//  Created by Anton Reynikov on 13.09.2023.
+//
+
+import UIKit
+
+final class PageViewController: UIViewController {
+    private let backgroundImage: UIImage
+    private let titlePage: String
+    private let descriptionPage: String
+    private let isThreePage: Bool
```

Comment:
Не совсем разобрался, "это три страницы" или "это третья страница"? Если второе, то можно поменять на 
```suggestion
    private let isThirdPage: Bool
```